react/components/organisms/listing/listing.js

```diff
@@ -0,0 +1,298 @@
+import React, { Component } from "react";
+import {
+    ActivityIndicator,
+    FlatList,
+    ScrollView,
+    StyleSheet,
+    Text,
+    View,
+    ViewPropTypes
+} from "react-native";
+import PropTypes from "prop-types";
+
+import { Search, Select } from "../../molecules";
+
+export class Listing extends Component {
+    static get propTypes() {
+        return {
+            items: PropTypes.array,

```

Comment:
Dead code?

Reply:
This is not dead code.
You can either use getItems to fetch data or items directly and use the callbacks to control listing refresh from the outside.
We will lose flexibility if we remove this.

Reply:
This listing must follow the same approach as the listing from ripe-components-vue. If you want to use a fixed set of items you can just `getItems={() => items}`